### ISSUE TITLE
Browser path in settings seems to be ignored

### ISSUE THREAD

**Issue by a-mckinley** [evilz/vscode-reveal#328]
I don't have Chrome installed at the default location of /usr/bin/google-chrome so I tried changing my Browser Path to '/usr/bin/firefox'.  This results in a failure to open the presentation in the browser.

The extension host logs shows the error message:

````
[exthost] [error] Error: No browser found
````

The only way I could make this work was add a symlink
```
sudo ln -s /usr/bin/google-chrome-stable /usr/bin/google-chrome
```

**Comment by evilz** [comment]
Hum, it should work. 
And it only works using chrome

**Comment by evilz** [comment]
Should use the default browser 
Fixed in [4.1.0 ](https://github.com/evilz/vscode-reveal/releases/tag/4.1.0)
